### DISCUSSION TITLE
[CSS]: Frontmatter items → Specific class instead of `#text`

### DISCUSSION THREAD

**Discussion by FelipeRearden** [tadashi-aikawa/obsidian-another-quick-switcher#188] [Ideas]
Hello @tadashi-aikawa 

There is one more thing related to CSS that I would like to share with you. 

For my workflow, **frontmatter items** are extremely important. Every note in my vault is filled with `aliases` `tags` and more metadata items.

Most of the time I use the **frontmatter items** to *navigate* and *search* inside an AQS modal.

📸 This is one of my AQS modals
![153970EF-475A-4EB6-A5A7-F59731B37624](https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/assets/85180696/8683713b-e410-4c01-b856-16463c099679)

## My FR

I would like to be able to target the **each element** `odd/even` of each frontmatter item.

📸 My FR
![959490E4-81E4-4B1C-99B7-69F9AAB315F2](https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/assets/85180696/00697db3-b33f-446d-a78f-f30631d3b94b)

Just as an example the CSS style, this is my note YAML frontmatter 

📸 YAML 
![F75ED05E-17D1-40A5-8643-8C2D770C7F9A](https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/assets/85180696/598414b9-1368-4f1b-8b6c-15055588d9c7)

ℹ️ In other words, my FR is about a specific class inside `.another-quick-switcher__item__meta__front_matter` that lets me use `:nth-of-type` to target `odd/even` items.

📸 Moreover apply the same behavior that we have for `.another-quick-switcher__item__description__link`  and `.another-quick-switcher__item__description__tag` 
![C36BD1BD-43E0-4ED7-88A2-2F568039A1A2](https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/assets/85180696/3bb9b95c-a586-4016-ac66-27035d3ef867)

## Why DO You Need This ?

The reason is that my eyes feels more comfortable with different colors. 

It is been hard for me to look/see each one of the items since they have the same color. 

That is the reason that my vault has a `font-size` much higher than the normal and my AQS modals are bigger and with a custom style. 

I need to make things comfortable for my 👁️

---

Thanks for reading this !!!!! I hope you like my suggestion 🙏 🙏 🙏

Let me know if you need more information !!!

If you decide to do this, I can create the FR :)

Have a great day 🌞

**Comment by tadashi-aikawa** [comment]
Hi, @FelipeRearden :)

LGTM. I converted to issue https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/issues/189 👍

**Reply by FelipeRearden** [reply]
Thank you very much @tadashi-aikawa !!!!! 🙏 🎉
